src/lib/utils.ts

```diff
@@ -0,0 +1,20 @@
+const fsLib = require('fs')
+const path = require('path');
+
+function collectDirFiles(dir: string): string[] {
+    let results: string[] = [];
+    let list: string[] = fsLib.readdirSync(dir);
+    list.forEach(function (file: string) {
+        file = path.join(dir, file);
+        let stat = fsLib.statSync(file);
+        if (stat && stat.isDirectory()) {
+            /* Recurse into a subdirectory */
+            results = results.concat(collectDirFiles(file));
+        } else {
+            /* Is a file */
+            results.push(file);
+        }
+    });
+    return results;
+}
```

Comment:
What if we leverage a bit of ES6 here?

```suggestion
const collectDirFiles = (dir: string): string[] =>
  fsLib.readdirSync(dir).reduce((results: string[], fileName: string) => {
    const fullFilePath = path.join(dir, fileName);
    const stat = fsLib.statSync(file);

    if (stat && stat.isDirectory()) {
      return [...results, ...collectDirFiles(fullFilePath)];
    }

    return [...results, fullFilePath];
  }, []);

```

Didn't test it though, please let me know if this works.